spec/models/ldap_spec.rb

```diff
@@ -17,6 +17,7 @@
   # rubocop:disable RSpec/ExampleLength
   describe '#find_by_netid' do
     it 'parses a valid ldap response' do
+      allow(described_class).to receive(:default_connection).and_return(ldap_connection)
```

Comment:
@tpendragon We should likely remove line 22 in favor of this line

Reply:
@carolyncole Done